vsintegration/src/FSharp.Editor/CodeLens/CodeLens.fs

```diff
@@ -122,7 +122,7 @@ type internal CodeLensTagger
         asyncMaybe {
             try
                 let! view = view
-                do! Async.Sleep 800 |> liftAsync
+                //do! Async.Sleep 5 |> liftAsync
```

Comment:
Intention here is to wait before we start refreshing the data. Refreshing is very costly. However let's disable it for the proof of concept